docs/self-hosted/self-hosted.md

```diff
@@ -10,9 +10,11 @@ Gitpod, just as you know it from [gitpod.io](https://gitpod.io), can be deployed
 
 ## Installation
 
-You can find all configuration templates and installation scripts in the Gitpod repository:
+You can find all the installation methods and relevant scripts under the [`install`](https://github.com/gitpod-io/gitpod/tree/master/install) directory.
 
-  > https://github.com/gitpod-io/gitpod
+**Recommended installation**: We strongly recommend choosing GCP or AWS to deploy and run a Gitpod instance. You can use our Terraform scripts to configure, deploy, and upgrade an instance.
+
+The state of deploying Gitpod on vanilla Kubernetes is not mature yet but you can try it by using the helm charts from the [`chart`](https://github.com/gitpod-io/gitpod/tree/master/chart) directory.
```

Comment:
**thought**: Got this from https://community.gitpod.io/t/multiple-installation-issues-0-5-0-vanilla-k3s-kubernetes/2221/2. Feel free to remove if you think this is redundant.

Reply:
I like the *recommended* part. But allthough the current documentation is lacking we still hope to have sth better before :christmas_tree: , and want to advertise "vanilla Kubernetes" as valid install option.

Reply:
Thanks for taking a look @geropl! WDYT of rephrasing the second part to something like the following?

```suggestion
Alternatively, you can use the helm charts from the [`chart`](https://github.com/gitpod-io/gitpod/tree/master/chart) directory to configure and deploy Gitpod on Kubernetes.
```

Reply:
For the 0.6.0 release the helm charts will be more mature so we don't need to recommend avoiding it.